### ISSUE TITLE
findConfig is a noop instead of throwing an error

### ISSUE THREAD

**Issue by tomzv** [browserslist/browserslist#826]
Hello, when creating a browserslist instance, the options object you can provide can include a  **path** property that provides, according to the README file -> file or a directory path to look for config file. 

When providing the **path** property in a browser environment, it just silently results in a noop when calling findConfig(), unlike when for example providing the **config** argument, (which throws an error when calling loadConfig() from the browser).

Wouldn't it be preferable to throw an error for the **path** property as well, to alert the user of the library of the fact that the **path** argument behaves differently in the browser? Is there a specific reason for the noop as opposed to throwing an error message?

**Comment by ai** [comment]
Nope. `postcss` for instance require `path` (to CSS file) and pass it to Browserslist always. Even in browser’s envrionment.

Also it will be a breaking change and release major updates is a huge problem for library like this.
